ci/run.sh

```diff
@@ -58,12 +58,15 @@ clone() {
 
 # build dmd (incl. building and running the unittests), druntime, phobos
 build() {
+    local unittest=${1:-1}
     if [ "$OS_NAME" != "windows" ]; then
         source ~/dlang/*/activate # activate host compiler, incl. setting `DMD`
     fi
     $DMD compiler/src/build.d -ofgenerated/build
-    generated/build -j$N MODEL=$MODEL HOST_DMD=$DMD DFLAGS="$CI_DFLAGS" BUILD=debug unittest
-    generated/build -j$N MODEL=$MODEL HOST_DMD=$DMD DFLAGS="$CI_DFLAGS" ENABLE_RELEASE=1 dmd
+    if [ $unittest -eq 1 ]; then
+        generated/build -j$N MODEL=$MODEL HOST_DMD=$DMD DFLAGS="$CI_DFLAGS" BUILD=debug unittest
+    fi
```

Comment:
The `unittest` command is disabled because it depends on `-Xf` to determine the version of dmd (added in 2.079/80, so earlier versions wrongly get detected as being gdc).

Reply:
Fine by me, the unittests aren't that crucial.